### DISCUSSION TITLE
Make asset background color configurable

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#4293] [Feature Requests]
Right now, resizing an image to a different aspect ratio using fit="contain" causes black bars to show up. This color should be configurable

**Comment by metalmarco** [comment]
More info about the issue here: https://github.com/directus/directus/issues/4292

**Comment by gdpdils** [comment]
This is much needed, indeed, and given that `sharp` already makes this possible, the implementation shouldn't take that long.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
